facade/nodeFacade.go

```diff
@@ -635,6 +635,16 @@ func (nf *nodeFacade) GetProofCurrentRootHash(address string) (*common.GetProofR
 	return nf.node.GetProof(hexRootHash, address)
 }
 
+func (nf *nodeFacade) getCurrentRootHash() []byte {
+	currentHeader := nf.blockchain.GetCurrentBlockHeader()
```

Comment:
is this method backwards compatible? Previously with or without nil currentHeader the nf.blockchain.GetCurrentBlockRootHash() was returned. Right now if the currentHeader is nil, it will return the nf.blockchain.GetLastExecutedBlockInfo().
It might not matter because it is on the nodeFacade...